python/tk_multi_workfiles/work_files.py

```diff
@@ -102,7 +102,17 @@ def show_file_open_dlg():
         handler = WorkFiles()
         from .file_open_form import FileOpenForm
 
-        handler._show_file_dlg("File Open", FileOpenForm)
+        handler._show_file_dlg("", FileOpenForm)
```

Comment:
Is that a mistake? It doesn't say File Open anymore.

Reply:
I just wanted to mess with people. I have no idea how that crept in, thanks!